vali_objects/vali_dataclasses/perf_ledger.py

```diff
@@ -148,13 +156,14 @@ def create_cps_to_fill_void(self, time_since_last_update_ms: int, now_ms: int, p
                                     prev_portfolio_ret=self.cps[-1].prev_portfolio_ret,
                                     prev_portfolio_spread_fee=self.cps[-1].prev_portfolio_spread_fee,
                                     prev_portfolio_carry_fee=self.cps[-1].prev_portfolio_carry_fee,
-                                    mdd=point_in_time_dd)
+                                    mdd=point_in_time_dd,
+                                    mpv=self.cps[-1].prev_portfolio_ret)
         assert new_cp.last_update_ms <= now_ms, self.cps
         self.cps.append(new_cp)
 
     def init_with_first_order(self, order_processed_ms: int, point_in_time_dd: float):
         new_cp = PerfCheckpointData(last_update_ms=order_processed_ms, prev_portfolio_ret=1.0,
-                                    mdd=point_in_time_dd, prev_portfolio_spread_fee=1.0, prev_portfolio_carry_fee=1.0)
+                                    mdd=point_in_time_dd, prev_portfolio_spread_fee=1.0, prev_portfolio_carry_fee=1.0, mpv=1.0)
```

Comment:
do we want to init mpv as 0.0?

Reply:
good question. Miners always begin with portfolio value of 1.0. so we want to init the max portfolio value to 1.0. 

I used 0.0 as an initial value elsewhere because that will allow the old perf ledgers to still be valid and mark those checkpoints as "legacy". Once all checkpoints are non-legacy, the new portfolio max calculation will be used.